charts/submodelservers/values.yaml

```diff
@@ -0,0 +1,138 @@
+# #############################################################################
+# Copyright (c) 2022,2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+# Copyright (c) 2021,2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# #############################################################################
+
+# Default values for submodelservers.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+nameOverride:
+fullnameOverride:
+
+image:
+  repository: ghcr.io/catenax-ng/catenax-at-home/provider-backend-service
```

Comment:
Is there nothing equivalent in Eclipse Tractus-X. 
If no, we should move this asap. The catenax-ng org will potentially be gone. 
The repo itself states, that it is out of date. There is also no 3rd party dependency check, which we would need to do for it.
So in my opinion, we cannot use this repo as dependency here

Reply:
I agree and tried to find something similar but it looks like this was not migrated to Eclipse Tractus-X.
The "proper" way would probably be to use one of the AASX Submodel Servers mentioned in the [Digital Twin Kit](https://eclipse-tractusx.github.io/docs-kits/kits/Digital%20Twin%20Kit/Operation%20View%20Digital%20Twin%20Kit#digital-twin-kit)
Since none of these Projects provide their own Helm Charts we cannot add them easily and I'm not familiar with the APIs and how to use them or integrate them with the existing test data upload script we were using in the IRS.

Reply:
Ok understood,

@stephanbcbauer, @danielmiehle, @Siegfriedk, @giterrific: What is your view from a project lead perspective?
Do we need a new repo/product, to pick up these missing things. I think we definitely need that for a proper way forward.

@AngelikaWittek: Do you see anything, that would prevent us from using the service provided by [catenax-ng/catenax-at-home](https://github.com/catenax-ng/catenax-at-home). I guess it as least proper IP checks right?

Reply:
@SebastianBezold is it about the image itself? We could fork the repo (no clue if this is legally okay for EF), we could clone the image into ours (to have it for test purposes only), we could fork the docker file and build it ourselves?

Reply:
Thank you for the insights. Since we only need a small docker image with create and get capabilities, we decided to implement this ourselves. As discussed with @jzbmw I prepared a PR to add this to this repo: https://github.com/eclipse-tractusx/tractus-x-umbrella/pull/54
I will remove the submodelservers chart form this branch. It will be added in scope of the other PR